content/source/docs/enterprise/admin/backup-restore.html.md

```diff
@@ -136,9 +173,7 @@ The JSON file used in the `config` form field should resemble the following:
 curl \
   --header "Authorization: Bearer $TOKEN" \
   --request POST \
-  --form config=@request.json \
+  --form config=@payload.json \
   --form snapshot=@backup.blob \
   https://<TFE HOSTNAME>/_backup/api/v1/restore
 ```
-
-~> **Note:** The Bearer token used to restore the backup is specific to the instance, so if restoring to a separate instance, the token will be different for restore as it was for backup.
```

Comment:
Mmm, I think we should keep a reminder about this inside the restore section... but it more properly belongs right at the START of the restore section. I'd put it right after the paragraph about "before starting, create an installation."

Basically we're just fallible humans out here, and I personally would be 100% guaranteed to forget about the difference in the authentication tokens somewhere between starting the backup and finally bringing up a fresh install.